packages/canon/src/css/utilities/height.css

```diff
@@ -0,0 +1,86 @@
+.cu-h {
+  height: var(--height);
+}
+
+.cu-min-h {
+  min-height: var(--min-height);
+}
+
+.cu-max-h {
+  max-height: var(--max-height);
+}
+
+/* Breakpoint xs */
+@media (min-width: 640px) {
+  .xs\:cu-h {
```

Comment:
How come these `\` are needed?

Reply:
`:` and`.` can't be use without a backslash on css. This pattern is commonly used with Tailwind, Radix, ... This would not be something we should use for common css classes but utility classes that's fine.

Reply:
ah alright, so it's actually part of the class name rather than a pseudo class